.rubocop.yml

```diff
@@ -195,3 +195,5 @@ Layout/InitialIndentation:
   Enabled: true
 Layout/LeadingCommentSpace:
   Enabled: true
+Metrics/AbcSize:
+  Enabled: false
```

Comment:
This should be in the `.rubocop_todo.yml` and can be generated by running `bundle exec rubocop --auto-gen-config`